app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+    cp = command_list[0]
+    f_in = command_list[1]
+    f_out = command_list[2]
+    if len(command_list) == 3 and cp == "cp":
```

Comment:
The input validation checks if the input string contains exactly three elements with `cp` as the first element, which is good. However, consider handling cases where the input string is malformed or has more than three elements.